src/App.tsx

```diff
@@ -1,7 +1,5 @@
 /* eslint-disable @typescript-eslint/no-misused-promises */
 import { useQuery } from '@tanstack/react-query';
-import fetchInstitutions from 'api/fetchInstitutions';
```

Comment:
A lot of diff in this file, but generally it's because I'm moving the `isUserEmailDomainAssociatedWithAnyInstitution` and `isEmailDomainAllowed` checks out to `ProfileForm`